src/backend/server.ts

```diff
@@ -60,16 +55,13 @@ backend(sessionConfig, prometheusTellere).then(({ app, azureAuthClient, router }
     // Sett opp bodyParser og router etter proxy. Spesielt viktig med tanke på større payloads som blir parset av bodyParser
     app.use(bodyParser.json({ limit: '200mb' }));
     app.use(bodyParser.urlencoded({ limit: '200mb', extended: true }));
+    // @ts-ignore
```

Comment:
Forslag på alternativ til denne?

Reply:
Sikker på den er nødvendig? Jeg får ingen warnings/feil 🤔

Reply:
Hmm, kjørte du `yarn`?

Reply:
Ja. slettet og `node_modules` først for å være sikker på at det ikke var gammel grums. Går også fint å bygge (`yarn build:prod`)

Reply:
Ja, jeg tror kanskje jeg hadde en rar versjon, fikk startet nå uten den ja 👍